src/app/context/ContactsProvider.tsx

```diff
@@ -0,0 +1,47 @@
+'use client';
+import { createContext, ReactNode, useEffect, useState } from 'react';
+import { ContactData, FormData } from '@/app/lib/types';
+import axios from 'axios';
+import { keysToTitleCase } from '@/app/lib/helpers';
+
+export const ContactsContext = createContext<FormData[]>([]);
+export const UpdateContext = createContext<() => void>(() => {});
+
+export default function ContactsProvider({
+  children,
+}: {
+  children: ReactNode;
+}): React.ReactNode {
+  const [contacts, setContacts] = useState<FormData[] | null>(null);
+  const [loading, setLoading] = useState(true);
+  const [state, setState] = useState<object>();
+  const forceUpdate = () => setState({});
+
+  useEffect(() => {
+    async function getContacts(): Promise<void> {
+      axios
+        .get('/api/contacts')
+        .then((res) => {
```

Comment:
make the argument name more meaningful
```suggestion
        .then((response/serverResponse) => {
```